### ISSUE TITLE
Console app args incorrect parsing

### ISSUE THREAD

**Issue by alex6dj** [dotnet/sdk#15075]
Im creating a console app in net 5.0. It just recursively copy a directory to another.

I just pass three args with each directory in this way.

Mainapp.exe "source" "target" "deleteTarget(boolean)"

The problem is when the source directory path ends with \"
Instead of 3 arguments I get 2. The second containing all the others.

A solution is to drop the slash at the end off the source path or use a double one \\"

In my case is a simple console app to copy a VS solution output after building. Because Im using $(TargetDir) it already put that simbol at the path end and fails doing his job.

Using net v5.0.100 and VS v16.8.2

**Comment by sfoslund** [comment]
> A solution is to drop the slash at the end off the source path or use a double one \"

Yes, this is expected. You will have to escape special characters if you want to use them as command line arguments. 

> Because Im using $(TargetDir) it already put that simbol at the path end and fails doing his job.

I'm not following what you mean here, can you provide more detail?

**Comment by alex6dj** [comment]
Thanks for your time. Next monday I will try to provide a project to show the problem
